module-cheatsheets/pathway-analysis-cheatsheet.md

```diff
@@ -0,0 +1,83 @@
+# Pathway Analysis Cheatsheet
+
+#### The tables below consist of valuable functions or commands that will help you through this module.
+##### Each table represents a different library/tool and its corresponding commands.
+> Please note that these tables are not intended to tell you all the information you need to know about each command.
+>
+> The hyperlinks found in each piece of code will take you to the documentation for further information on the usage of each command.
+
+<div style="page-break-after: always;"></div>
+
+### `msigdbr`
+
+Read the `msigdbr` documentation [**here**](https://www.rdocumentation.org/packages/msigdbr/versions/7.1.1).
+
+| Library/Package| Piece of Code| What it's called| What it does |
+|----------------|--------------|-----------------|--------------|
+| `msigdbr`| [`msigdbr()`](https://www.rdocumentation.org/packages/msigdbr/versions/7.1.1/topics/msigdbr)| geom vertical line| Retrieves the specified MSigDB dataset |
+
+### AnnotationDbi 
+Read the `AnnotationDbi` package vignette [**here**](http://bioconductor.org/packages/release/bioc/vignettes/AnnotationDbi/inst/doc/IntroToAnnotationPackages.pdf).
+
+| Library/Package                                  | Piece of Code                               | What it's called                               | What it does                                                                                                                                   |
+|-----------------------------------------|---------------------------------------------|-------------------------|------------------------------------------------------------------------|
+| `AnnotationDbi`        | [`keytypes()`](https://www.rdocumentation.org/packages/AnnotationDbi/versions/1.44.0/topics/AnnotationDb-objects)      | Keytypes     | Returns a character vector of column names/keytypes (eg., type of gene identifiers) available in an `AnnotationDbi` package.                                              |
+| `AnnotationDbi`       | [`mapIDs()`](https://www.rdocumentation.org/packages/AnnotationDbi/versions/1.44.0/topics/AnnotationDb-objects) | Mapped IDs       | Extracts the mapped ids for a set of keys (e.g., gene identifiers) of a specific keytype                                    |  
```

Comment:
The words `keys`, `keytypes`, and `columns` are kinda ambiguous/confusing for this. Can we put this in more plain language? May want to reference [Jackie's wording about mapIds here](https://github.com/AlexsLemonade/training-modules/blob/4e191a2fb621f6826299b5d02c8566d3dd6eb484/pathway-analysis/01-overrepresentation_analysis.Rmd#L161).